### PR TITLE
Iterating on landing pages flakiness

### PR DESCRIPTION
**Pull request by hannahbergam**
The first update is to change the url in one of the files to go straight to the redirected link. This is slightly speculative, but it is where the test consistently fails and I suspect Saucelabs is having a tough time with the cross-domain test.

Additionally, I'm disabling the second video resources test for now, as it's still blocking folks who are dotd and we would see any universal YT integration failure in the other landing page tests.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by kelbyhawn — APPROVED** [review]
🎉 🤞

File: dashboard/test/ui/features/acquisition_products/pegasus_landing_pages_2.feature [hunk 1]
```diff
@@ -15,5 +15,3 @@ Examples:
   | http://code.org/athletes                                          | athletes tutorial landing  |
   | http://code.org/tools/applab                                      | app lab tutorial landing   |
   | http://code.org/dance                                             | dance tutorial landing     |
-  | http://code.org/educate/resources/videos                          | video resources landing    |
-#  | http://code.org/educate/resources/videos?force_youtube_fallback=1 | resources yt fallback      |
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/acquisition_products/pegasus_landing_pages_2.feature:19]: Can we talk out loud in sprint planning about removing these?

Comment (hannahbergam) [reply dashboard/test/ui/features/acquisition_products/pegasus_landing_pages_2.feature:19]: Yes of course! I also know we'll need to address the YouTube header issue as well.

Comment (hannahbergam) [reply dashboard/test/ui/features/acquisition_products/pegasus_landing_pages_2.feature:19]: After what we talked about in sprint planning, I'm going to merge this PR as-is and prioritize working with Kelby on the YT titles and adding ignore regions as @dmcavoy requested to maintain maximum coverage on other pages.

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**